packages/now-cli/test/dev-server.unit.js

```diff
@@ -498,3 +498,12 @@ test('[DevServer] parseListen()', t => {
   t.truthy(err);
   t.is(err.message, 'Unknown `--listen` scheme (protocol): bad:');
 });
+
+test(
+  '[DevServer] index.html prioritized over other index.* files',
+  testFixture('now-dev-prioritize-index-html', async (t, server) => {
```

Comment:
Let's move this test to `test/dev/integration.js` so we can compare with production.

We also need to test that `/index.css` returns the correct content too.

Take a look at this this test for example: https://github.com/vercel/vercel/blob/8ec8affe29e49c27c437d6d74daa8a4f343fce94/packages/now-cli/test/dev/integration.js#L472-L482